torch_xla/csrc/ops/amin.cpp

```diff
@@ -0,0 +1,49 @@
+#include "torch_xla/csrc/ops/amin.h"
+
+#include "tensorflow/compiler/xla/xla_client/util.h"
+#include "torch_xla/csrc/lowering_context.h"
+#include "torch_xla/csrc/ops/infer_output_shape.h"
+#include "torch_xla/csrc/reduction.h"
+
+namespace torch_xla {
+namespace ir {
+namespace ops {
+namespace {
+
+xla::Shape NodeOutputShape(const Value& input,
+                           std::vector<xla::int64>& dimensions, bool keepdim) {
+  auto lower_for_shape_fn =
+      [&](absl::Span<const xla::XlaOp> operands) -> xla::XlaOp {
+    return BuildMinInDims(operands[0], dimensions, keepdim);
+  };
+  return InferOutputShape({input.shape()}, lower_for_shape_fn);
+}
+
+}  // namespace
+
+Amin::Amin(const Value& input, std::vector<xla::int64> dimensions, bool keepdim)
+    : Node(ir::OpKind(at::aten::argmax), {input},
```

Comment:
Can you fix it to amin?

Reply:
@JackCaoG 
We commonly use capitalize class names under ops. Is there a reason we should use `amin` here?

Reply:
This is the Node kind instead of OP name, that will be displayed when you dump the ir graph. We do not own most of these `aten` Node kinds, they are defined by pytorch.

```
(pytorch2) jackcao@jackcao64:~/Desktop/gitProj2/pytorch/aten/src/ATen/core$ grep 'logsumexp' *
interned_strings.h:  _(aten, logsumexp)                 \
interned_strings.h:  _(aten, special_logsumexp)         \
```

However I did not find the symbol for `amax` and `amin`. You might need to submit a pr like https://github.com/pytorch/pytorch/pull/36851 to pytroch first. @bdhirsh can help you with that.

Reply:
I found `at::aten::argmax/argmin` defined [here](https://github.com/pytorch/pytorch/blob/2fd37a830ede2c01e498ed103f62fd3421c01b5d/aten/src/ATen/core/aten_interned_strings.h#L194). But from the docs, it looks like [`argmin`](https://pytorch.org/docs/stable/generated/torch.argmin.html) and [`min`](https://pytorch.org/docs/stable/generated/torch.amin.html) are different, so it sounds reasonable to add them as symbols. I can put up a PR

Reply:
(https://github.com/pytorch/pytorch/pull/61550)

Reply:
Thanks @bdhirsh